CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### 💫 New checkers
 
+- Added new checker that checks f-string to see if it only consists of a single bare format expression that can be replaced with the string representation of that expression
```

Comment:
This entry should start with the name of the error being checked (see past entries for "New checkers")